service/history/ndc/transaction_manager.go

```diff
@@ -187,6 +187,7 @@ func NewTransactionManager(
 			shardContext,
 			workflowCache,
 			logger,
+			false,
```

Comment:
Could use a comment explaining why this is false.

Reply:
So we plan to keep the logic the same for replication stack?